__mocks__/solid-auth-client.ts

```diff
@@ -1,5 +1,5 @@
 export default {
-  currentSession: function() {
+  currentSession: function () {
     return Promise.resolve('http://w.e/b#id')
   }
-}
```

Comment:
this I would have opened and saved this file before pushing the time before and didn't run the linter